internal/tool/certs/certs.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certs
```

Comment:
What do you think about moving this package to `/internal/certs` instead of `/internal/tool/certs`?

Reply:
I'm not super thrilled to move it to internal/certs, since the certificates generated/verified here are pretty crappy (e.g., self-signed CA, fake organization info, certificates with a fixed duration etc.). These are meant for in-memory deployers like multi or weavertest, and for nothing that touches production (e.g., weavelet, GKE). I don't want to give folks an impression that the weavelet uses these in any way.

Reply:
Totally reasonable, makes sense.